src/commands/mods-configure.ts

```diff
@@ -48,10 +48,12 @@ export default new Command("mods:configure <instanceId>")
       const paramSpecWithNewDefaults = paramHelper.getParamsWithCurrentValuesAsDefaults(
         existingInstance
       );
+      const currentLocation = _.get(existingInstance, "configuration.params.LOCATION");
       const params = await paramHelper.getParams(
         projectId,
         paramSpecWithNewDefaults,
-        options.params
+        options.params,
+        currentLocation
```

Comment:
I'm not sure that adding this parameter to `getParams` is the correct choice. Yes, it works, but it's adding a very specific "if" to the logic in `getParams` (which actually has to be plumbed even further down to `.ask`, indicating to me that this _really_ isn't great). It's a specific option for way too specific of a case.

How about this as an alternative? `paramSpecWithNewDefaults` returns an array of params - what if the `LOCATION` param was removed from that list before `paramSpecWithNewDefaults` was passed into `getParams`? That way, `getParams` is simply getting params for what it's asked for, and we don't have to pass down a current location.

It may be the case then, that the `location` param has to be re-added back to `params` when it comes back from `getParams`, but that should be pretty easy to do as well.

What do you think?

Reply:
I bounced around between a few implementations of this, and I was pretty torn. I was trying to keep as much param manipulation logic out of the command, to keep things clean and easier to unit test. I tried something similar to what you suggested, but inside paramHelper, and it ended up as a mess, so I moved toward this solution.

Having said that, I just rewrote this with the param 
logic in the command, and its much cleaner, so I agree with your suggestion.

Here's another pic of the output of configure and install after making this change, still looks good:
<img width="1435" alt="Screen Shot 2019-07-30 at 4 03 08 PM" src="https://user-images.githubusercontent.com/4635763/62171473-dc9b5b00-b2e3-11e9-9337-0f0e49f1c42f.png">